datasets/recipes/compressed_archive.json

```diff
@@ -0,0 +1,49 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
+    "sc": "https://schema.org/",
+    "ml": "http://mlcommons.org/schema/",
+    "includes": "ml:includes",
+    "recordSet": "ml:RecordSet",
+    "field": "ml:Field"
+  },
+  "@type": "sc:Dataset",
+  "name": "Compressed archive example",
+  "description": "This is a fairly minimal example, showing a way to describe archive files.",
+  "url": "https://example.com/datasets/recipes/compressed_archive/about",
+  "distribution": [
+    {
+      "name": "archive",
```

Comment:
I don't know if that is a consensus, but I think we should name the files using the same filename as much as possible, here: "compressed_archive.zip".

This makes it easier to follow later, but also would make it easier for libraries such as TFDS or HF to save the file with the most correct name on disk, especially when filename cannot be inferred from URL. It would also help in cases where resources need to be downloaded manually (there are a few in TFDS for example) because of various reasons (user needs to approve license, etc.).

Reply:
I think that's an excellent suggestion! Fixed in https://github.com/mlcommons/datasets_format/pull/37/commits/0553950c6f24146b7f5c9fba88abf92c7d738144